src/View/DynamicContentAwareTrait.php

```diff
@@ -18,7 +18,7 @@ trait DynamicContentAwareTrait
      *
      * @return View the view object that can be used to render views or view files.
      */
-    abstract protected function getView(): View;
+    abstract protected function getView(): WebView;
```

Comment:
Do we rely on something that is `WebView`-specific when using this method?

Reply:
yes, return WebView passed by the constructor.

Reply:
Would View work instead?

Reply:
In the WebView you have all the functions of the assets + view, in the view you only have view.